frontend/src/store/questions/index.js

```diff
@@ -16,17 +20,9 @@ const state = {
 }
 
 const getters = {
-  feed: ({questions}) => {
-    return questions.map(({ questionID, userID, title, body }) => ({
-      questionID,
-      title,
-      body,
-      author: userID,
-      published: format(Date.now(), 'DD/MM/YY')
-    }))
-  },
-  question: ({questionObj: {question}}) => question,
-  answers: ({questionObj: {answers}}) => answers,
+  feed: ({questions}) => questions.map(questionMapper),
```

Comment:
Again, I think 'feed' isn't descriptive enough, because it could refer to many things. 'questionFeed' or 'questions' is better imo

Reply:
yep